app/org/maproulette/models/dal/ChallengeDAL.scala

```diff
@@ -520,6 +524,44 @@ class ChallengeDAL @Inject() (
     }
   }
 
+  def updateSpatialFields(user: User)(implicit challengeId: Long, c: Option[Connection] = None) = {
```

Comment:
This is a lot of computation and it will be a concern if this executes on every challenge update (eg a challenge-admin changes the description) or if tasks are added to an existing challenge (eg "bulk task add").

Is the result of the "isGlobal" required immediately or can it wait for a scheduled task to execute? We have a task that updates a challenge's task bounding, and my though is that this computation for "isglobal" could be done in that task (and, in fact, it could look at the challenge's just-computed bounds and not re-compute).

Reply:
either one is an option
>(and, in fact, it could look at the challenge's just-computed bounds and not re-compute).

if we can do that, that would be nice